<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDock_options</class>
 <widget class="QDialog" name="LogDock_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>356</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>TextEdit</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="cb_readonly">
          <property name="text">
           <string>только для чтения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_10">
          <property name="text">
           <string>цветные логи</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_11">
          <property name="text">
           <string>не добавлять CRLF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_12">
          <property name="text">
           <string>добавлять дату и время</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_13">
          <property name="text">
           <string>отображать ошибки, как MsgBox</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Codecs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>CodecForCStrings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_CodecForCStrings"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Отображать логи:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="cb_EMERG">
          <property name="text">
           <string>EMERG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_ALERT">
          <property name="text">
           <string>ALERT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_CRIT">
          <property name="text">
           <string>CRIT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_ERR">
          <property name="text">
           <string>ERR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_WARN">
          <property name="text">
           <string>WARN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_NOTICE">
          <property name="text">
           <string>NOTICE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_INFO">
          <property name="text">
           <string>INFO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_DEBUG">
          <property name="text">
           <string>DEBUG</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
